Pay period schema

teegolden opened the pull request:
Feature:
As an admin, so I can see the pay period I am scheduling for, I want to see only the two corresponding weeks of a Google calendar

TODO:
a) Incorporate this to the google calendar after @stevenbuccini  and @MegSesh are done.

b)Associate pay period and Shifts after @stevenbuccini create a CRUD interface the shift model or this get assign to someone else on next iteration. 

c)Write cucumber test after a and b

sepkristina commented:
I'm having a hard time seeing how this will work with the google calendar. How does the admin access a previous pay period to send out urgent emails, is it that same 'edit' button that we saw yesterday? Steven recommended not having a pay period model and instead just scrape the calendar for current open shifts. I think we need to have a group convo about how exactly this should work.

sepkristina commented:
what happens when you press the edit pay period button? I think this could work if when you push that button, it leads you to another page that is full of radio buttons of current pay periods, and then when you click a radio button, it displays the google cal and the unfilled shifts for that pay period, do you see what i'm saying?

teegolden commented:
@sepkristina that what its doing. Except I haven't add the calendar, and will not be added till next iteration.

teegolden commented:
Ok I just fixed some merge conflicts. Some of @sepkristina test is relying on methods I changed, that is fixed now. I will go ahead and merge this now. If anyone have a comment feel free to still make comments.
